simple/securedrop-sdk/index.html

```diff
@@ -21,5 +21,7 @@ <h1>Links for securdrop-sdk</h1>
     <a href="/localwheels/securedrop_sdk-0.3.0-py3-none-any.whl">securedrop_sdk-0.3.0-py3-none-any.whl</a><br/>
     <a href="/localwheels/securedrop-sdk-0.3.1.tar.gz">securedrop-sdk-0.3.1.tar.gz</a><br/>
     <a href="/localwheels/securedrop_sdk-0.3.1-py3-none-any.whl">securedrop_sdk-0.3.1-py3-none-any.whl</a><br/>
+    <a href="/localwheels/securedrop-sdk-0.3.1.tar.gz">securedrop-sdk-0.3.2.tar.gz</a><br/>
```

Comment:
The tarball link also needs updating

Reply:
twice in a row! sorry bout that. that's it i'm creating an automation ticket for this step

Reply:
:boom: https://github.com/freedomofpress/securedrop-debian-packaging/issues/273